### ISSUE TITLE
Coverage Drops After Incorporating Translator v3.14.0

### ISSUE THREAD

**Issue by jkotanchik-SB** [projecttacoma/fqm-execution#313]
### Summary

After updating to the 3.14.0 cql-to-elm Translator, our QA team identified reduced Coverage values on some hand-picked measures:

| # | Measure | 3.14.0 | 3.3.2 |
| - | --------- |-----| -------|
|1| CMS135FHIR | 99% | 100% |
|2| CMS144FHIR | 96% | 100% |
|3| CMS145FHIR | 99% | 100% |
|4| CMS159FHIR | 99% | 100% |
|5| CMS645FHIR | 94% | 99% |
|6| CMS646FHIR | 95% | 99% |
|7| CMS771FHIR | 94% | 100% |

#### External Tracking Ticket
N/A

### Expected Behavior

Coverage matches when using older and current cql-to-elm version.

### Version or Commit
`v1.5.0`

### Inputs (e.g. Measure Bundle, Patient Bundle, CQL Library)
[MeasureBundles.zip](https://github.com/user-attachments/files/17077786/MeasureBundles.zip)

Measure Exports:
1. [CMS145FHIR-v0.2.000-FHIR4.zip](https://github.com/user-attachments/files/17561456/CMS145FHIR-v0.2.000-FHIR4.zip)
2. [CMS72FHIR-v0.7.000-FHIR4.zip](https://github.com/user-attachments/files/17561459/CMS72FHIR-v0.7.000-FHIR4.zip)
3. [CMS1028FHIR-v0.1.000-FHIR4.zip](https://github.com/user-attachments/files/17561460/CMS1028FHIR-v0.1.000-FHIR4.zip)

Patient Bundles:
1. [CMS135FHIR-v0.2.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077736/CMS135FHIR-v0.2.000-FHIR4-TestCases.zip)
3. [CMS144FHIR-v1.4.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077743/CMS144FHIR-v1.4.000-FHIR4-TestCases.zip)
4. [CMS145FHIR-v0.2.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077739/CMS145FHIR-v0.2.000-FHIR4-TestCases.zip)
5. [CMS159FHIR-v0.2.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077745/CMS159FHIR-v0.2.000-FHIR4-TestCases.zip)
6. [CMS645FHIR-v1.4.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077747/CMS645FHIR-v1.4.000-FHIR4-TestCases.zip)
7. [CMS646FHIR-v1.4.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077749/CMS646FHIR-v1.4.000-FHIR4-TestCases.zip)
8. [CMS771FHIR-v1.4.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17077751/CMS771FHIR-v1.4.000-FHIR4-TestCases.zip)
9. [CMS72FHIR-v0.7.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17561464/CMS72FHIR-v0.7.000-FHIR4-TestCases.zip)
10. [CMS1028FHIR-v0.1.000-FHIR4-TestCases.zip](https://github.com/user-attachments/files/17561463/CMS1028FHIR-v0.1.000-FHIR4-TestCases.zip)

### Relevant Calculation Options (e.g. Measurement Period, meta.profile Validation)
```
{
    includeClauseResults: false,
    trustMetaProfile: true,
    buildStatementLevelHTML: true,
    measurementPeriodStart: 1/1/2025,
    measurementPeriodEnd: 12/31/2025,
}
```
### cql-to-elm Version Used for Measure Logic Translation (if known)
`v3.14.0`

### Any Additional Info
Similar to #285, with the updated cql-to-elm translator, the highlighting is showing aliases uncovered.

**Comment by elsaperelli** [comment]
Hi @jkotanchik-SB ! Could you provide us with the previous measure bundles that used the 3.3.2 version of the translator? Thank you!

**Comment by jkotanchik-SB** [comment]
Re-built the Measure bundles using the 3.3.2 Translator. There might be some minor differences due to having to pull the measure content from PROD into my local to complete the rebuild. (For instance, random whitespace in the CQL so that I could enable the save changes button to complete the rebuild in MADiE.)

[MeasureBundles-3_3_2.zip](https://github.com/user-attachments/files/17248139/MeasureBundles-3_3_2.zip)

**Comment by lmd59** [comment]
Hi [@jkotanchik-SB](https://github.com/jkotanchik-SB),
We are in process for addressing your issue with PR https://github.com/projecttacoma/fqm-execution/pull/314, and we'd like to be able to more thoroughly test against all of the measures you've provided with issues here. Unfortunately, we've hit a snag with testing the 3.14.0-translated measure CMS145 that you've provided, where there's a syntax error in the JSON (`SyntaxError: Unterminated string in JSON at position 241835`). Maybe something has gone wrong with this bundle's creation, and the end of the bundle json was cut off. Do you have a properly formatted version of this measure bundle that you'd be able to share with us?

**Comment by lmd59** [comment]
Hi [@jkotanchik-SB](https://github.com/jkotanchik-SB), I just wanted to bump the above comment in case it got lost in the mix! Let me know if you have another version of the CMS145 measure. Thanks!

**Comment by jkotanchik-SB** [comment]
@lmd59 
Oh, deary me! It really did get lost in the inbox. I've added the Measure Export for CMS145FHIR, which includes the bundle as the top level JSON file.

We recently had a couple more cases of coverage dropping after we updated to fqm-execution 1.5.0. The drops may still be related to the changes in Translator 3.14.0, so I added those bundles to this ticket for your review. Please let me know if it would be better to open another Issue.

CMS72 and CMS1028 both claim to have been at 100% coverage prior to MADiE moving to fqm-execution 1.5.0.

**Comment by lmd59** [comment]
@jkotanchik-SB 
Thanks so much for the updated and new measures for testing! I was able to confirm that the PR at https://github.com/projecttacoma/fqm-execution/pull/314 has returned both CMS145 and CMS72 to 100% coverage for the 3.14.0 translator update. For CMS1028, I'm seeing 99.7% coverage (for each group) with translator v3.3.2, then dropping to 99.5% coverage with the translator update to v3.14.0. The linked PR returns this measure to 99.7% coverage.

**Comment by lmd59** [comment]
Resolved with [v1.6.0](https://github.com/projecttacoma/fqm-execution/releases/tag/v1.6.0)
